code/dynamic_programming/src/edit_distance/edit_distance.js

```diff
@@ -0,0 +1,45 @@
+// dynamic programming | edit distance | Javascript
+// Part of Cosmos by OpenGenus Foundation
+
+editDistance = (str1, str2) => {
+    const lenStr1 = str1.length;
+    const lenStr2 = str2.length;
+    
+
+    if (lenStr1 == 0) {
+        return lenStr2;
+    }
+
+    if (lenStr2 == 0) {
+        return lenStr1;
+    }
+
+    let distanceVectorInit = [];
+    let distanceVectorFinal = [];
+
+    for(var i=0; i<=lenStr1; i++) {
```

Comment:
```suggestion
    for(let i=0; i <= lenStr1; i++) {
```

Please do this for all other loops too.